analyses/malawi/dryspells_trigger/exploration/compare_arc2_chirps.Rmd

```diff
@@ -0,0 +1,483 @@
+  coord_sf(datum = NA) +
+  theme_minimal() +
+  labs(title = "Dry spells only detected using touching") +
+  theme(plot.title.position = "plot")
+
+plot_grid(centroid_missing_plot, touch_missing_plot)
+```
+
+### Trigger activation
+
+Let's consider if we're using these datasets for trigger activation and how they would
+compare. We will only consider dry spells that are confirmed at some point in January
+and February. Let's just quickly look at the number of dry spells detected this time period
+for each method to start.
+
+```{r trigger-chirps}
+chirps_df %>%
+  filter(month(chirps_confirmation) <= 2) %>%
+  mutate(trigger_year = year(chirps_confirmation)) %>%
+  group_by(trigger_year) %>%
+  summarize(n_triggers = n(),
+            .groups = "drop")
+```
+
+```{r trigger-center}
+arc2_center_df %>%
+  filter(month(arc2_center_confirmation) <= 2) %>%
+  mutate(trigger_year = year(arc2_center_confirmation)) %>%
+  group_by(trigger_year) %>%
+  summarize(n_triggers = n(),
+            .groups = "drop")
+```
+
+```{r trigger-touch}
+arc2_touch_df %>%
+  filter(month(arc2_touch_confirmation) <= 2) %>%
+  mutate(trigger_year = year(arc2_touch_confirmation)) %>%
+  group_by(trigger_year) %>%
+  summarize(n_triggers = n(),
+            .groups = "drop")
+```
+
+Given that the 2 ARC2 methods are so similar, we likely only need to look at the centroid method.
+This is because based on the matrix above, there are only 6 dry spells detected through
+the touch method not detected through the centroid method, but all but one of those
+is a city area. Alternatively, the centroid method would result in 8 triggers not captured
+by the touch method. However, only one of these is an urban area and the other 7
+are south-central districts in the southern region.
+
+```{r get-rid-of-touch}
+arc2_chirps_df %>%
+  filter(arc2_touch_ds, !arc2_centroid_ds) %>%
+  transmute(ADM2_EN, trigger_year = year(arc2_touch_confirmation))
+```
+
+```{r keep-that-centroid}
+arc2_chirps_df %>%
+  filter(!arc2_touch_ds, arc2_centroid_ds) %>%
+  transmute(ADM2_EN, trigger_year = year(arc2_center_confirmation))
+```
+
+Although the centroid method fails to capture Zomba City (due to a lack of raster
+centroid for ARC2 data), it seems sufficient to consider vs. CHIRPS excluding the
+touching raster method.
+
+### Daily value comparisons
+
+First, let's do some 2d histograms to examine the distribution of various calculation methods.
+What do we see between the centroid and touching raster methods applied?
+
+```{r 2d-histo-arc}
+daily_df %>%
+  ggplot(aes(x = arc2_centroid,
+             y = arc2_touch)) +
+  geom_bin2d(bins = 200) +
+  theme_bw() +
+  geom_abline(intercept = 0)
+```
+
+While it looks like the touching method might skew higher (if just slightly) at
+higher levels of rainfall, we are more concerned at their behavior at lower levels.
+
+```{r 2d-histo-arc-1}
+daily_df %>%
+  filter(arc2_centroid <= 1 | arc2_touch <= 1) %>%
```

Comment:
Sorry maybe I'm not totally following here, but how is it that the axes on the figure below extend past 1 if you're filtering for precipitation values <= 1?

Reply:
I'm filtering to instances where at least one of the measurements is `<= 1`, so those values are showing large deviations for a measurement compared to the other.

Reply:
Ah I see, so where ARC2 would be 0.2 but CHIRPS could be something like 100, for example?